README.md

```diff
@@ -110,15 +110,49 @@ namespace FileUploader
 ```
 
 ## Running Minio Client Examples
+#### On Windows 
 * Clone this repository.
 
-* Build the project to produce the Minio.Examples console app.
+* Build the project in Visual Studio to produce the Minio.Examples console app.
 
 * Move into Minio.Examples directory and enter your credentials and bucket name, object name etc.
   Uncomment the example test cases such as below in Program.cs to run an example.
 ```cs
   //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
 ```
+* Run the Minio.Client.Examples.NET452 or Minio.Client.Examples.NetCore project from Visual Studio
+#### On Linux (Ubuntu 16.04 and above)
+* Clone this repository.
+
+* Move into Minio.Examples directory and enter your credentials and bucket name, object name etc.
```

Comment:
This part has to be re-written to make it simpler @poornas .. @NitishT can give his inputs here as well.